pkg/transformers/namespace.go

```diff
@@ -44,6 +44,9 @@ var skipNamespacePathConfigs = []PathConfig{
 		GroupVersionKind: &schema.GroupVersionKind{
 			Kind: "ClusterRoleBinding",
 		},
+		// The namespace pointed by this Path will be updated
+		Path:               []string{"subjects", "namespace"},
+		CreateIfNotPresent: false,
```

Comment:
Should this be `true`?  What about cases where the user wishes to include a clusterrolebinding subject from a separate namespace? (also discussed this in https://github.com/kubernetes-sigs/kustomize/pull/156)